platform/services/account/app/common/utils/logger.go

```diff
@@ -32,18 +29,12 @@ func syncLogLevel(loggerLevel *zap.AtomicLevel) {
 		"error":   zapcore.ErrorLevel,
 	}
 	for {
-		loggingConfigDir := os.Getenv(loggingConfigDirEnvVarName)
```

Comment:
The idea behind using this file to store log's level was to allow application to react to change of this level without a need of restarting pods - as only when you mount something as a file - its content is changed when the source is changed. Thanks this application may react if you change the source log level.
It doesn't work for env variables - as they remain unchanged for the whole life of a container - so to apply log's level change there - you need to restart the pod.
Thus I would suggest to stick to the current mechanism.